timings/state.go

```diff
@@ -116,13 +115,21 @@ func flattenRecursive(data *rootTimingsJSON, timings []*Span, nestLevel int, max
 	}
 }
 
-// Save appends Timings data to the "timings" list in the state and purges old timings, ensuring
-// that up to MaxTimings are kept. Timings are only stored in state if their duration is greater
-// than or equal to DurationThreshold.
-// It's responsibility of the caller to lock the state before calling this function.
-func (t *Timings) Save(st *state.State) {
+// A GetSaver helps storing Timings (ignoring their details).
+type GetSaver interface {
+	GetTimings(timings interface{}) error
```

Comment:
I think I should call this one GetMaybeTimings to be consistent with other places that use GetMaybe for when the data being absent is ok (no error)

Reply:
+1